src/SourceLocator/Type/Composer/PsrAutoloaderLocator.php

```diff
@@ -54,11 +55,11 @@ public function locateIdentifier(Reflector $reflector, Identifier $identifier):
     /**
      * Find all identifiers of a type
      *
-     * @return list<Reflection>
+     * @return Generator<Reflection>
      */
-    public function locateIdentifiersByType(Reflector $reflector, IdentifierType $identifierType): array
+    public function locateIdentifiersByType(Reflector $reflector, IdentifierType $identifierType): Generator
```

Comment:
The implementation of this class can be rolled back, except for the return type being `iterable`